### ISSUE TITLE
AppStore rejection because of "PAY TOTAL" string in the ApplePay Sheet

### ISSUE THREAD

**Issue by DmIvanov** [Shopify/mobile-buy-sdk-ios#781]
Our app was rejected with such an issue: "Your app uses Apple Pay as a purchasing mechanism but does not use Apple Pay branding and user interface elements appropriately as described in the Apple Pay Human Interface Guidelines. Specifically, the Apple Pay payment sheet does not provide details regarding who the users is paying."

After the investigation of https://developer.apple.com/ios/human-interface-guidelines/technologies/apple-pay/ and a phone conversation with Apple it turned out that thr company/store name is needed to be mention in that line ("PAY TOTAL").

![img_3304](https://user-images.githubusercontent.com/3761598/32370318-e2c292d2-c08c-11e7-9145-5931df20bf0d.jpg)

As a quick fix to pass the review we've simply changed the string in PayCheckout -> summaryItems: [PKPaymentSummaryItem]
from
```
summaryItems.append(self.paymentDue.summaryItemNamed("TOTAL"))
```
to
```
summaryItems.append(self.paymentDue.summaryItemNamed("<OUR COMPANY>"))
```

After resubmission the app was approved.

Can we expect it to be implemented in future releases of the SDK? 
(It seems like your colleagues on web have already changed it and when using ApplePay from web checkout the sheet looks correct - with a company/store name)

**Comment by dbart01** [comment]
Thanks for bringing this to our attention. It will definitely be fixed in a future release.

**Comment by cocoahero** [comment]
Just had the same rejection notice. A quick fix would be appreciated.

**Comment by dbart01** [comment]
Now fixed in [v3.1.3](https://github.com/Shopify/mobile-buy-sdk-ios/releases/tag/3.1.3)
